libs/design-system/logos/apps/src/lib/logos-apps.component.html

```diff
@@ -0,0 +1,13 @@
+<a class="hra-logo" href="https://humanatlas.io" target="_blank" rel="noopener noreferrer">
+  <img [attr.src]="'logo/hra_small.svg' | assetUrl" alt="Human Reference Atlas home" class="hra-small" />
```

Comment:
Update src path to `'assets/logo/hra_small.svg' | assetUrl`. I've reconfigured storybook to put assets in the in the assets folder instead of the root folder